ast/src/object/display.rs

```diff
@@ -56,11 +56,17 @@ impl Display for Instr {
 }
 
 impl<T: Display> Display for LinkTo<T> {
+    fn fmt(&self, f: &mut Formatter<'_>) -> Result {
+        write!(f, "// {} in {}", self.function, self.machine)
+    }
+}
+
+impl Display for Machine {
     fn fmt(&self, f: &mut Formatter<'_>) -> Result {
         write!(
             f,
-            "// {} in object with latch \"{}\" and function_id \"{}\"",
-            self.function, self.latch, self.function_id
+            "object with latch \"{}\" and function_id \"{}\"",
```

Comment:
```suggestion
            "// object with latch \"{}\" and function_id \"{}\"",
```
?

Reply:
No, it's the second part of the `LinkTo` format

Reply:
sorry, I'm still a bit lost. Does this go in a comment line or is this actual code?

Reply:
it goes in the second part of a comment line for LinkTo
// operation foo in **object with latch ...**

Reply:
ok!